src/main/scala/no/ndla/imageapi/service/ImportService.scala

```diff
@@ -83,11 +83,11 @@ trait ImportService {
           val key = imageMeta.mainImage.originalFile
           val image = domain.Image(key, imageMeta.mainImage.originalSize.toInt, imageMeta.mainImage.originalMime)
 
-          val imageMetaInformation = domain.ImageMetaInformation(None, titles, alttexts.flatten, image.url, image.size, image.contentType, copyright, tags, captions.flatten)
+          val imageMetaInformation = domain.ImageMetaInformation(None, titles, alttexts.flatten, image.fileName, image.size, image.contentType, copyright, tags, captions.flatten)
 
-          if (!imageStorage.contains(key)) imageStorage.uploadFromUrl(image, key, sourceUrlFull)
+          if (!imageStorage.objectExists(key)) imageStorage.uploadFromUrl(image, key, sourceUrlFull)
```

Comment:
Njei. Egentlig ikke. Vi lener oss på at filnavnet er det samme i enkelte tilfeller. (F.eks. denne bildesjekkeren som Johannes snakket om vil kunne ha god bruk for det). Å generere et nytt navn vil derfor kunne bryte den koblingen.

Tror kanskje det riktige faktisk er å overskrive bildefilen i de tilfellene. (Og kanskje bruke noe prefix el. til å sørge for at genererte filnavn aldri treffer noe de har brukt i eksisterende løsning). Tar gjerne en kommentar fra @kstigen også på dette.